qhub/template/{{ cookiecutter.repo_directory }}/image/jupyterhub/environment.yaml

```diff
@@ -9,4 +9,5 @@ dependencies:
  - escapism
  - pip:
    - jupyterhub-kubespawner >= 0.16.1
-   - qhub-jupyterhub-theme
+   - https://github.com/Quansight/qhub-jupyterhub-theme/archive/b78749053d5ddd094c684cf48a0b4e227880ea65.tar.gz
```

Comment:
I'd prefer if we cut releases for these and pinned them.

Reply:
I'll create a release of qhub-jupyterhub-theme. Is it too early to have a cdsdashboard release?

Reply:
It needs more testing for a full public release, but I can do a beta release on PyPI if that makes you more comfortable?

Reply:
Should use cdsdashboards==0.5.0-b1. I can update both when you've done a qhub theme release (or feel free to do it yourself if quicker for you).

Reply:
Now pinned to new versions